assessment.sql

```diff
@@ -5,24 +5,53 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
-
+-- CREATE TABLE licensees(
+  --in: create_table/licensees.sql
+id SERIAL PRIMARY KEY,
+surname CHARACTER VARYING,
+given_name CHARACTER VARYING,
+gender CHARACTER (1),
+height INTEGER,
+weight INTEGER,
+dob DATE,
+eye_color CHARACTER VARYING,
+hair_color CHARACTER VARYING
+);
+--in psql --in psql: \i scripts/create_table/licensees.sql
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
-
--- your code here
+--in insert_data/licensees.sql--
+INSERT INTO licensees(surname,given_name, gender, height, weight, dob, eye_color, hair_color)
+VALUES (`Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black`);
```

Comment:
You've surrounded the entire comma-separated list of values with backticks, which will not communicate any information about the type of the data you're inserting to the database. Strings need to be in single quotes, numbers should be unquoted, and dates must be specified as strings lest they be parsed as an integer expression. 1/2